schema.json

```diff
@@ -14,24 +19,19 @@
         "type": "INTEGER",
         "mode": "NULLABLE"
     },
-    {
-        "name": "timestamp",
-        "type": "STRING",
-        "mode": "NULLABLE"
-    },
     {
         "name": "blockTimestamp",
-        "type": "STRING",
+        "type": "TIMESTAMP",
         "mode": "NULLABLE"
     },
     {
-        "name": "msg",
+        "name": "rawEvent",
         "type": "STRING",
         "mode": "NULLABLE"
     },
     {
         "name": "blockGasUsed",
-        "type": "INT",
```

Comment:
Do you know if there a difference? I changed all `INT` to `INTEGER` because when partitioning a table, bigquery can partition by `INTEGER` but not `INT`, which is weird, but I just changed it and didn't investigate further